apollo-backend/src/main/java/io/logz/apollo/notifications/senders/MarkersSender.java

```diff
@@ -0,0 +1,100 @@
+    private final String X_API_TOKEN_HEADER = "X-API-TOKEN";
+    private final String apiToken;
+    private final String createMarkersEndpointUrl;
+
+    public MarkersSender(String notificationJsonConfiguration) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        DeploymentMarkersConfiguration markersConfiguration = mapper.readValue(notificationJsonConfiguration, DeploymentMarkersConfiguration.class);
+        this.apiToken = markersConfiguration.getApiToken();
+        this.createMarkersEndpointUrl = markersConfiguration.getCreateMarkersEndpointUrl();
+    }
+
+    @Override
+    public boolean send(NotificationTemplateMetadata notificationTemplateMetadata) {
+        OkHttpClient httpClient = new OkHttpClient();
+        Request.Builder builder = new Request.Builder().url(this.createMarkersEndpointUrl)
+                .header(X_API_TOKEN_HEADER, this.apiToken);
+
+        DeploymentMarkers deploymentMarkers = getDeploymentMarkers(notificationTemplateMetadata);
+        executeMarkersRequest(httpClient, builder, deploymentMarkers);
+        return true;
+    }
+
+    private void executeMarkersRequest(OkHttpClient httpClient, Request.Builder builder, DeploymentMarkers deploymentMarkers) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        try {
+            String markersJson = objectMapper.writeValueAsString(deploymentMarkers);
+
+            Request request = builder.post(RequestBody.create(MediaType.parse("application/json; charset=utf-8"), markersJson)).build();
+            try (okhttp3.Response response = httpClient.newCall(request).execute()) {
+                if (HttpStatus.isPositive(response.code())) {
+                    logger.info("successfully sent marker");
+                } else {
+                    logger.warn("exception while sending marker");
+                }
+            } catch (IOException e) {
+                logger.warn("IO Exception when trying to send deployment markers request", e);
+            }
+        } catch (JsonProcessingException jsonProcessingException) {
+            logger.warn("Could not process json", jsonProcessingException);
+        }
+    }
+
+    private DeploymentMarkers getDeploymentMarkers(NotificationTemplateMetadata deployment) {
+        DeploymentMarker deploymentMarker = new DeploymentMarker();
+        deploymentMarker.setTitle(deployment.getServiceName());
+        deploymentMarker.setDescription(String.format("Deployment to service: %s, with commit message: %s", deployment.getServiceName(), deployment.getDeploymentMessage()));
+
+        HashMap<String, String> metadata = new HashMap<>();
+        metadata.put("deployment_message", String.valueOf(deployment.getDeploymentMessage()));
+        metadata.put("deployment_id", String.valueOf(deployment.getDeploymentId()));
+        metadata.put("deployment_phase", String.valueOf(deployment.getStatus()));
```

Comment:
I think it's more suitable for it to be deploymnet_status and not deployment_phase :)